### ISSUE TITLE
Angular module missing from nativescript-fonticon 8.1.0

### ISSUE THREAD

**Issue by sublime392** [nstudio/nativescript-ui-kit#26]
I get this error after updating to version 8.1.0. Problem is temporarily resolved by pinning to version ~8.0.2.

`Module not found: Error: Can't resolve 'nativescript-fonticon/angular'`

**Comment by NathanWalker** [comment]
I’ll check in a bit, could be the module type in the /angular/package. When possible, mind sharing what you see on your end, can copy/paste contents here (code node_modules/nativescript-fonticon/angular/package.json)

**Comment by brianrclow** [comment]
I'm getting the same error as well.

```Successfully synced application dev.launchpoint.launchpointapp on device 00008101-000818C12250001E.
  ***** Fatal JavaScript exception - application has been terminated. *****
  NativeScript encountered a fatal error:
  Error calling module function
  Error: Cannot find module 'nativescript-fonticon/angular'```

Here is a screenshot, there is no package.json in node_modules/nativescript-fonticon/angular/package.json. 
<img width="1331" alt="image" src="https://github.com/nstudio/nativescript-ui-kit/assets/22532259/4dbd6572-01d6-4206-bdbb-35e0de3d1404">

Also is there any reason this plugin isn't under @nstudio scope like the others? Just curious.

**Comment by NathanWalker** [comment]
Resolved in 8.1.2, lemme know if further issue.

**Comment by NathanWalker** [comment]
@brianrclow there was still some debate as to whether we just build this directly into core and so name stayed flattened for now. I think it probably should be built into core in future.
